e2e-tests/src/test/validators_change.rs

```diff
@@ -28,7 +27,7 @@ pub fn change_validators(config: &Config) -> anyhow::Result<()> {
 
     accounts.remove(0);
     let new_members: Vec<AccountId> = accounts.iter().map(|pair| pair.public().into()).collect();
-    send_change_members(&connection, new_members.clone());
+    change_members(&connection, new_members.clone(), XtStatus::InBlock);
```

Comment:
Nice, passing everywhere now `XtStatus`, so it's clear for the developer what's the block waiting status. Good standard in e2e test code.